packages/react-redux/src/hooks/use-all-feature-toggles/use-all-feature-toggles.ts

```diff
@@ -1,30 +1,20 @@
-import { getIsFeatureEnabled, useAdapterContext } from '@flopflip/react';
-import { type TFlags } from '@flopflip/types';
+import { type TAdapterIdentifiers, type TFlags } from '@flopflip/types';
 import { useSelector } from 'react-redux';
 
 import { selectFlags } from '../../ducks/flags';
 
-export default function useAllFeatureToggles(): TFlags {
+export default function useAllFeatureToggles(
+  adapterIdentifiers: TAdapterIdentifiers
+): TFlags {
   const allFlags = useSelector(selectFlags());
-  const adapterContext = useAdapterContext();
 
-  const enabledFlags: TFlags = Object.values(allFlags).reduce<TFlags>(
-    (_enabledFlags, flags) => {
-      Object.keys(flags).forEach((featureName) => {
-        if (
-          getIsFeatureEnabled(
-            allFlags,
-            adapterContext.adapterEffectIdentifiers,
-            featureName
-          )
-        ) {
-          _enabledFlags[featureName] = true;
-        }
-      });
-      return _enabledFlags;
-    },
-    {}
-  );
+  let allFlagsFromAdapterIdentifers: TFlags = {};
```

Comment:
I was hesitant about returning all the flags or only the enabled ones.
Is there any difference, from the consumer point of view, between checking for a flag that does not exist against a flag that exists but it's not enabled?
I mean, if the behaviour is the same, reading the disabled flags seem useless to me.

Reply:
There is no difference. A disabled flag (or any variation) is the same as a non existent flag. However, if we again think in terms of export/import for this style of hook you might want to know if the value is actually defined vs undefined. Especially for none-boolean variate flags. Cause there you have to pass the variation to determine if it matches. As we never want to pass all variations to return booleans for a variation only returning enabled flags quickly becomes a bit odd.